src/applications/vaos/services/appointment/transformers.js

```diff
@@ -358,159 +358,48 @@ function setLocation(appt) {
-
-/**
- * Builds contained array and populates with video conference info and facility info if available
- *
- * @param {Object} appt  VAR appointment object
- * @returns {Array|Object} Returns array of contained objects of FHIR appointment containing video conference info or
- * returns an object containing patient information.
- */
-function setContained(appt) {
-  switch (getAppointmentType(appt)) {
-    case APPOINTMENT_TYPES.request: {
-      return createPatientResourceFromRequest(appt);
-    }
-    case APPOINTMENT_TYPES.ccRequest: {
-      let contained = createPatientResourceFromRequest(appt);
-      if (appt.ccAppointmentRequest?.preferredProviders) {
-        appt.ccAppointmentRequest.preferredProviders.forEach(
-          (provider, index) => {
-            const address = [];
-            if (provider.address) {
-              address.push({
-                line: [provider.address?.street],
-                city: provider.address?.city,
-                state: provider.address?.state,
-                postalCode: provider.address?.zipCode,
-              });
-            }
-
-            contained = {
-              ...contained,
-              practitioner: {
-                id: `cc-practitioner-${appt.id}-${index}`,
-                name: provider.lastName
-                  ? {
-                      text: `${provider.firstName} ${provider.lastName}`,
-                      family: provider.lastName,
-                      given: provider.firstName,
-                    }
-                  : null,
-                address: provider.address ? address : null,
-                practitionerRole: [
-                  {
-                    location: [
-                      {
-                        reference: `Location/cc-location-${appt.id}-${index}`,
-                        display: provider.practiceName,
-                      },
-                    ],
-                  },
-                ],
-              },
-            };
-          },
-        );
-      }
-      return contained;
-    }
-    case APPOINTMENT_TYPES.ccAppointment: {
-      let address;
-      if (appt.address) {
-        address = {
-          line: [appt.address.street],
-          city: appt.address.city,
-          state: appt.address.state,
-          postalCode: appt.address.zipCode,
-        };
-      }
-
       return {
-        practitioner: {
-          resourceType: 'Location',
-          id: `cc-location-id`,
-          name: appt.providerPractice,
-          address: appt.address ? address : null,
-          telecom: appt.providerPhone
-            ? [
-                {
-                  system: 'phone',
-                  value: appt.providerPhone,
-                },
-              ]
-            : null,
-        },
+        vistaId: appt.facility?.facilityCode?.substring(0, 3),
+        stationId: appt.facility?.facilityCode,
       };
     }
-    case APPOINTMENT_TYPES.vaAppointment:
     default:
-      return null;
+      return {};
```

Comment:
Make sure we always have a `location` object on the appointment, even if it's empty